### PR TITLE
[1623] Update uk and international fee data type

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
UK and International Fees

### Changes proposed in this pull request
Specifical set UK and International fees to integer

### Guidance to review
Post should not be a string

### CODE REVIEW

**Comment by dankmitchell** [comment]
I don't think we actually need this change 🤔

**Comment by benilovj** [comment]
It seems like this change will improve our model of the underlying data?

**Comment by tvararu** [comment]
@dankmitchell is it because we have frontend validations on the input? We should disable those I think.

**Comment by dankmitchell** [comment]
[Add missing validations to enrichments](https://trello.com/c/e7wGJYfZ/1623-add-missing-validations-to-enrichments)

File: spec/requests/courses/fees_spec.rb [hunk 1]
```diff
@@ -105,8 +105,8 @@
 
         original_course_details = [
                                     course_2.course_length,
-                                    course_2.fee_uk_eu,
-                                    course_2.fee_international
+                                    course_2.fee_uk_eu.to_s,
```
Comment (defong) [inline spec/requests/courses/fees_spec.rb]: you sure you need to to string it?

Comment (dankmitchell) [reply spec/requests/courses/fees_spec.rb]: JSON API Client 🤷‍♂

**Comment by fofr** [comment]
I think this has introduced this bug:
https://trello.com/c/weZDU1qy/1671-saving-an-empty-fee-gets-converted-to-0

**Pull request merged**